src/DeviceInput/deviceInputSystem.ts

```diff
@@ -138,10 +138,10 @@ export class DeviceInputSystem implements IDisposable {
         }
 
         if (!this._inputs[deviceType][deviceSlot]) {
-            const device = new Array<Nullable<number>>(numberOfInputs);
+            const device = new Array<number>(numberOfInputs);
 
             for (let i = 0; i < numberOfInputs; i++) {
-                device[i] = null;
+                device[i] = 0; /* set device input as unpressed */

```

Comment:
So the original reason why we kept this as null was that there wasn't a single initialization value that worked for every device.  While gamepads, keyboards, and mouse buttons makes sense to have a init value of 0, mouse/touch x and y don't always start at 0.  This would mean that there would be a brief window where the pointer's location would be incorrect but still be a valid value (as opposed to null, which could be checked and ignored).  If we go this route, it might also be a good idea to immediately update the values without triggering an onInputChanged event.

Reply:
This could be easily done by adding
```
const pointer = this._inputs[deviceType][deviceSlot];
pointer[0] = evt.clientX;
pointer[1] = evt.clientY;
```
to just after the _registerDevice calls for both _pointerMoveEvent and_pointerDownEvent.

Reply:
I can dig it. I'll add this into this PR as well.